src/workerd/io/worker-entrypoint.c++

```diff
@@ -324,10 +326,13 @@ kj::Promise<void> WorkerEntrypoint::request(kj::HttpMethod method,
     TRACE_EVENT_END("workerd", PERFETTO_TRACK_FROM_POINTER(&context));
     TRACE_EVENT("workerd", "WorkerEntrypoint::request() run", PERFETTO_FLOW_FROM_POINTER(this));
     jsg::AsyncContextFrame::StorageScope traceScope = context.makeAsyncTraceScope(lock);
+    jsg::Ref<api::AbortSignal> signal =
+        abortController.emplace(jsg::alloc<api::AbortController>())->getSignal();
+    signal->setIgnoredForSubrequests();
```

Comment:
I'd generally prefer if this were set as an argument via the `api::AbortController` constructor as there's really not much reason to change it once set.